internal/service/cloudbackupsnapshotrestorejob/data_source_cloud_backup_snapshot_restore_job.go

```diff
@@ -35,8 +35,9 @@ func DataSource() *schema.Resource {
 				Computed: true,
 			},
 			"created_at": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:       schema.TypeString,
+				Computed:   true,
+				Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.18.0"),
```

Comment:
[part of old v1 api](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v1/#tag/Cloud-Backups/operation/listBackupExportJobs)

removed in [v2](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Cloud-Backups/operation/getBackupRestoreJob)